### ISSUE TITLE
HifiGan finetuning doesn't work

### ISSUE THREAD

**Issue by jeffersonquispe** [NVIDIA/NeMo#4276]
**Describe the bug**
Have the following bug when run commamd to traininig HifiGan Finetuning:
Error in call to target 'nemo.collections.tts.torch.data.VocoderDataset':
FileNotFoundError(2, 'No such file or directory')
full_key: train_ds.dataset

**Steps/Code to reproduce bug**

command as the turorial : 
python hifigan_finetune.py --config-name=hifigan.yaml model.train_ds.dataloader_params.batch_size=32 model.max_steps=1000 model.optim.lr=0.00001 ~model.optim.sched rain_dataset=./mex_manifest_train_dur_180_mins_local_mels/hifigan_train_ft.json validation_datasets=./mex_manifest_val_dur_180_mins_local_mels/hifigan_val_ft.json exp_manager.exp_dir=hifigan_ft +init_from_pretrained_model=tts_hifigan trainer.check_val_every_n_epoch=10 model/train_ds=train_ds_finetune model/validation_ds=val_ds_finetune

**Expected behavior**

Start training HifiGan Finetuning

**Environment overview (please complete the following information)**

 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider - AWS, Azure, GCP, Collab)]
 - Method of NeMo install: [pip install or from source]. Please specify exact commands you used to install.
 - If method of install is [Docker], provide `docker pull` & `docker run` commands used

**Environment details**
pip list:
absl-py                       1.0.0
aiohttp                       3.8.1
aiosignal                     1.2.0
alabaster                     0.7.12
aniso8601                     9.0.1
antlr4-python3-runtime        4.8
appdirs                       1.4.4
argon2-cffi                   21.3.0
argon2-cffi-bindings          21.2.0
asttokens                     2.0.5
async-timeout                 4.0.2
attrdict                      2.0.1
attrs                         21.4.0
audioread                     2.1.9
Babel                         2.10.1
backcall                      0.2.0
beautifulsoup4                4.11.1
black                         19.10b0
bleach                        5.0.0
boto3                         1.23.8
botocore                      1.26.8
braceexpand                   0.1.7
cachetools                    5.1.0
certifi                       2022.5.18.1
cffi                          1.15.0
charset-normalizer            2.0.12
click                         8.1.3
colorama                      0.4.4
cycler                        0.11.0
Cython                        0.29.30
debugpy                       1.6.0
decorator                     5.1.1
defusedxml                    0.7.1
Distance                      0.1.3
docker-pycreds                0.4.0
docopt                        0.6.2
docutils                      0.17.1
editdistance                  0.6.0
entrypoints                   0.4
executing                     0.8.3
fastjsonschema                2.15.3
fasttext                      0.9.2
filelock                      3.7.0
Flask                         2.1.2
Flask-RESTful                 0.3.9
fonttools                     4.33.3
frozendict                    2.3.2
frozenlist                    1.3.0
fsspec                        2022.5.0
ftfy                          6.1.1
g2p-en                        2.1.0
gdown                         4.4.0
gitdb                         4.0.9
GitPython                     3.1.27
google-auth                   2.6.6
google-auth-oauthlib          0.4.6
grpcio                        1.46.3
h5py                          3.7.0
huggingface-hub               0.7.0
hydra-core                    1.1.2
idna                          3.3
imagesize                     1.3.0
importlib-metadata            4.11.4
inflect                       5.6.0
iniconfig                     1.1.1
ipadic                        1.0.0
ipykernel                     6.13.0
ipython                       8.3.0
ipython-genutils              0.2.0
ipywidgets                    7.7.0
isort                         4.3.21
itsdangerous                  2.1.2
jarowinkler                   1.0.2
jedi                          0.18.1
jieba                         0.42.1
Jinja2                        3.1.2
jmespath                      1.0.0
joblib                        1.1.0
jsonschema                    4.5.1
jupyter-client                7.3.1
jupyter-core                  4.10.0
jupyterlab-pygments           0.2.2
jupyterlab-widgets            1.1.0
kaldi-io                      0.9.4
kaldi-python-io               1.2.2
kaldiio                       2.17.2
kiwisolver                    1.4.2
latexcodec                    2.0.1
librosa                       0.9.1
llvmlite                      0.38.1
Markdown                      3.3.7
MarkupSafe                    2.1.1
marshmallow                   3.15.0
matplotlib                    3.5.2
matplotlib-inline             0.1.3
mecab-python3                 1.0.5
mistune                       0.8.4
mpmath                        1.2.1
multidict                     6.0.2
nbclient                      0.6.3
nbconvert                     6.5.0
nbformat                      5.4.0
nemo-toolkit                  1.8.1
nest-asyncio                  1.5.5
nltk                          3.7
notebook                      6.4.11
numba                         0.55.2
numpy                         1.22.4
oauthlib                      3.2.0
omegaconf                     2.1.2
onnx                          1.11.0
OpenCC                        1.1.3
packaging                     21.3
pandas                        1.4.2
pandocfilters                 1.5.0
pangu                         4.0.6.1
parameterized                 0.8.1
parso                         0.8.3
pathspec                      0.9.0
pathtools                     0.1.2
pesq                          0.0.4
pexpect                       4.8.0
pickleshare                   0.7.5
Pillow                        9.1.1
pip                           20.3.4
pip-api                       0.0.29
pipreqs                       0.4.11
pkg-resources                 0.0.0
pluggy                        1.0.0
pooch                         1.6.0
portalocker                   2.4.0
prometheus-client             0.14.1
promise                       2.3
prompt-toolkit                3.0.29
protobuf                      3.20.1
psutil                        5.9.1
ptyprocess                    0.7.0
pure-eval                     0.2.2
py                            1.11.0
pyannote.core                 4.4
pyannote.database             4.1.3
pyannote.metrics              3.2
pyasn1                        0.4.8
pyasn1-modules                0.2.8
pybind11                      2.9.2
pybtex                        0.24.0
pybtex-docutils               1.0.2
pycparser                     2.21
pyDeprecate                   0.3.2
pydub                         0.25.1
Pygments                      2.12.0
pyparsing                     3.0.9
pypinyin                      0.46.0
pyrsistent                    0.18.1
PySocks                       1.7.1
pystoi                        0.3.3
pytest                        7.1.2
pytest-runner                 6.0.0
python-dateutil               2.8.2
pytorch-lightning             1.6.3
pytz                          2022.1
PyYAML                        5.4.1
pyzmq                         23.0.0
rapidfuzz                     2.0.11
regex                         2022.4.24
requests                      2.27.1
requests-oauthlib             1.3.1
resampy                       0.2.2
rsa                           4.8
ruamel.yaml                   0.17.21
ruamel.yaml.clib              0.2.6
s3transfer                    0.5.2
sacrebleu                     2.1.0
sacremoses                    0.0.53
scikit-learn                  1.1.1
scipy                         1.8.1
Send2Trash                    1.8.0
sentencepiece                 0.1.96
sentry-sdk                    1.5.12
setproctitle                  1.2.3
setuptools                    59.5.0
shellingham                   1.4.0
shortuuid                     1.0.9
simplejson                    3.17.6
six                           1.16.0
smmap                         5.0.0
snowballstemmer               2.2.0
sortedcontainers              2.4.0
SoundFile                     0.10.3.post1
soupsieve                     2.3.2.post1
sox                           1.4.1
Sphinx                        4.5.0
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-bibtex          2.4.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        2.0.0
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.5
stack-data                    0.2.0
sympy                         1.10.1
tabulate                      0.8.9
tensorboard                   2.9.0
tensorboard-data-server       0.6.1
tensorboard-plugin-wit        1.8.1
terminado                     0.15.0
threadpoolctl                 3.1.0
tinycss2                      1.1.1
tokenizers                    0.12.1
toml                          0.10.2
tomli                         2.0.1
torch                         1.11.0+cu113
torch-stft                    0.1.4
torchaudio                    0.11.0+cu113
torchmetrics                  0.8.2
torchvision                   0.12.0+cu113
tornado                       6.1
tqdm                          4.64.0
traitlets                     5.2.1.post0
transformers                  4.19.2
typed-ast                     1.5.4
typer                         0.4.1
typing-extensions             4.2.0
Unidecode                     1.3.4
urllib3                       1.26.9
wandb                         0.12.17
wcwidth                       0.2.5
webdataset                    0.1.62
webencodings                  0.5.1
Werkzeug                      2.1.2
wget                          3.2
wheel                         0.37.1
widgetsnbextension            3.6.0
wrapt                         1.14.1
yarg                          0.1.9
yarl                          1.7.2
youtokentome                  1.0.6
zipp                          3.8.0
More details
- Debian GNU/Linux
- PyTorch version 1.11.0+cu113
- Python version 3.9

**Additional context**
GPU Testla T4 nvidia

**Comment by redoctopus** [comment]
It doesn't look like it can find your dataset. Can you double check the location?

**Comment by mllopartbsc** [comment]
Hi, I'm finding the same problem.
